lib/client/signalflow/signalflow_client.js

```diff
@@ -63,28 +62,78 @@ function SignalflowClient(apiToken, options) {
         return metaDataMap[tsid];
       },
       stream: function (fn, errorFn) {
-        if (typeof fn !== 'function') {
+        if (typeof fn !== "function") {
           return false;
         }
         callback = fn;
         msgBuffer.forEach(resolveMessage);
         msgBuffer = [];
         return true;
-      }
+      },
     };
   }
 
+  function liveTailRequest(opts) {
+    var msgBuffer = [];
+    var callback = null;
+
+    function resolveMessage(msg) {
+      // we're duck typing error messages here, but it should be okay.
+      if (msg.hasOwnProperty("error") && !msg.type) {
```

Comment:
I don't think your message definition ever defines that field / error messages, does it?